modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -0,0 +1,394 @@
+    --cidr 0.0.0.0/0
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID --protocol "tcp" \
+    --port 30081 \
+    --cidr 0.0.0.0/0
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID --protocol "tcp" \
+    --port 30282 \
+    --cidr 0.0.0.0/0
+# Adjust the port 32094 if the Redpanda broker count is not 3.
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID \
+    --protocol "tcp" \
+    --port 32092-32094 \
+    --cidr 0.0.0.0/0
+# Adjust the port 35084 if the Redpanda broker count is not 3.
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID \
+    --protocol "tcp" \
+    --port 35082-35084 \
+    --cidr 0.0.0.0/0
+----
+
+=== Create VPC subnet
+
+Run the following command, specifying the subnet availability zone (for example, `usw2-az1`). You need the subnet ID `subnet_id` from the command output to <<create-vpc-endpoint,create a VPC endpoint>>.
+
+[,bash]
+----
+aws ec2 create-subnet --region $REGION --profile $PROFILE --vpc-id $CLIENT_VPC_ID \
+    --availability-zone <zone> \
+    --cidr-block 10.0.1.0/24
+SUBNET_ID=<subnet_id>
+----
+
+=== Create VPC endpoint
+
+[,bash]
+----
+aws ec2 create-vpc-endpoint \
+    --region $REGION --profile $PROFILE \
+    --vpc-id $CLIENT_VPC_ID \
+    --vpc-endpoint-type "Interface" \
+    --ip-address-type "ipv4" \
+    --service-name $PL_SERVICE_NAME \
+    --subnet-ids $SUBNET_ID \
+    --security-group-ids $SECURITY_GROUP_ID \
+    --private-dns-enabled
+----
+
+== Access Redpanda services through VPC endpoint
+
+You can access Redpanda services such as Schema Registry and HTTP proxy from the client VPC, for example, from an EC2 instance in the VPC.
+
+=== Access Kafka API seed service
+
+Use port `30292` to access the Kafka API seed service.
+
+[,bash]
+----
+export REDPANDA_BROKERS=rp-endpoint.$CLUSTER_DOMAIN:30292
+rpk cluster info --tls-enabled --user <user> --password <password>
+----
+
+If successful, the `rpk` output should look like the following:
+
+[,bash,role=no-copy]
+----
+CLUSTER
+=======
+redpanda.rp-cki01qgth38kk81ard3g
+
+BROKERS
+=======
+ID    HOST                                                                PORT   RACK
+0*    0-3da65a4a-0532364.cki01qgth38kk81ard3g.fmc.dev.cloud.redpanda.com  32092  use2-az1
+1     1-3da65a4a-63b320c.cki01qgth38kk81ard3g.fmc.dev.cloud.redpanda.com  32093  use2-az1
+2     2-3da65a4a-36068dc.cki01qgth38kk81ard3g.fmc.dev.cloud.redpanda.com  32094  use2-az1
+----
+
+=== Access Schema Registry seed service
+
+Use port `30081` to access the Schema Registry seed service.
+
+[,bash]
+----
+curl -vv -u <user>:<password> -H "Content-Type: application/vnd.schemaregistry.v1+json" --sslv2 --http2 https://rp-endpoint.$CLUSTER_DOMAIN:30081/subjects
+----
+
+=== Access Redpanda Proxy seed service
```

Comment:
@paulzhang97 Just to confirm, does this refer to HTTP Proxy (formerly pandaproxy)?

Reply:
Yes